content/docs/command-reference/plots/index.md

```diff
@@ -25,311 +25,11 @@ positional arguments:
 
 ## Description
 
-DVC provides a set of commands to visualize data produced by machine learning
-projects. Usual plots include AUC curves, loss functions, or confusion matrices,
-for example. Plots are a great alternative to `dvc metrics` when working with
-multi-dimensional performance data. They also help you present and compare
-[experiments] effectively.
+`dvc plots` subcommands configure and visualize data and images produced by
+machine learning projects.
```

Comment:
I see this follows the pattern in the [`exp` ref](https://dvc.org/doc/command-reference/exp#description). That can work 👍🏼 

But what about going with something like the [`cache`](https://dvc.org/doc/command-reference/cache) and  [`stage`](https://dvc.org/doc/command-reference/stage#description) refs instead? That is: a brief paragraph around the concept in question, then one with a summary of each subcommand.

I'm honestly not sure which pattern to go with but it may be a good time to decide (we'll need it again to finish https://github.com/iterative/dvc.org/issues/2572).

For now, quick suggestion on existing text:
```suggestion
`dvc plots` subcommands help you visualize and compare data or images
produced by machine learning projects, letting you fully customize graph
settings and style.
```